playwright.config.ts

```diff
@@ -34,12 +34,12 @@ const config: PlaywrightTestConfig = {
     /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
     actionTimeout: 0,
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: `http://localhost:${process.env.PORT}`,
+    baseURL: `http://localhost:${process.env.PORT || 8080}`,
```

Comment:
Since we are using port 8080 everywhere, I think it would be better to just change this to use a hard coded port
```suggestion
    baseURL: 'http://localhost:8080',
```